src/components/UI/NetworkSwap/NetworkSwap.module.scss

```diff
@@ -1,9 +1,8 @@
 .networkSwap {
   position: absolute;
-  margin: -20px auto auto auto;
   z-index: 1;
   left: 50%;
-  transform: translate(-50%);
+  transform: translate(-50%, calc(-50% - 2px));
```

Comment:
why?

Reply:
to center it (instead of the margin -20px used before). the 2px is to consider the separate line